crates/factor-outbound-http/src/wasi_2023_10_18.rs

```diff
@@ -8,9 +8,7 @@ mod bindings {
 
     wasmtime::component::bindgen!({
         path: "../../wit",
-        interfaces: r#"
-            include wasi:http/proxy@0.2.0-rc-2023-10-18;
-        "#,
+        world: "wasi:http/proxy@0.2.0-rc-2023-10-18",
```

Comment:
Do you recall why we needed the inline interface before? (and don't now)

Reply:
IIRC `world` only accepted "foo" before and not "foo:bar/baz". I also think that historically there wasn't a `wasi:http/proxy` world here but it was an amalgamation of other things. EIther that or I copy/pasted from elsewhere which had these motivations and this specific place didn't. Either way though I believe the two are equivalent and `interfaces: ...` was generating an error I wasn't quite sure why but `world` seemed to fix it.